biblatex-iso690.tex

```diff
@@ -374,16 +467,55 @@ \subsubsection{Specific entry types}
 \t|type| field to specify the type of the thesis -- a localization
 term or literal string can be entered. For the list of supported
 localization terms, please refer to section 4.9.2.13 of the
-\t|biblatex| documentation. Names of the supervisor and school
-(institution) can be entered into the fields \t|supervisor| and
-\t|institution|, respectively.
+\t|biblatex| documentation. Available terms are \t|bathesis|,
+\t|mathesis|, \t|phdthesis|, and \t|candthesis|. Names of the
+supervisor and school (institution) can be entered into the fields
+\t|supervisor| and \t|institution|, respectively.
+\newline
+
+\c{Luptak2016thesis}
+\begin{ltxexample}
+@THESIS{Luptak2016thesis,
+  author       = "Lupták, Dávid",
+  title        = "Typesetting of Bibliography According to ISO 690 Norm",
+  date         = 2016,
```

Comment:
Integer-only values are special and can drop the `"..."` or `{...}` but since `date` may contain more than just integers, I think it is good practice to add the `{...}` even if it is not needed. 

```suggestion
  date         = {2016},
```